pop-cli-for-appchains/guides/calling-a-chain.md

```diff
@@ -0,0 +1,72 @@
+# Call a chain
+
+### Interactive Guidance (Recommended)
+
+Interact with the chain **with** Pop CLI's interactive guidance by simply entering:
+
+```shell
+pop call chain
+```
+
+You will be prompted to select a pallet, choose a dispatchable function, specify any required arguments, then provide connection details and the information of the account that will sign the extrinsic.
+
+<figure><img src="../.gitbook/assets/callchain.gif" alt="pop call chain"><figcaption><p>pop call chain</p></figcaption></figure>
+
+### Manual (non-interactive)
+
+If you prefer not to use interactive prompts, you can call the chain by specifying all the required arguments directly. For example, to call a dispatchable function on a specific pallet:
+
+```shell
+pop call chain --pallet System --function remark --args "0x11" --url ws://localhost:9944/ --suri //Alice
+```
+
+```
+┌   Pop CLI : Call a chain
+│
+◇  Would you like to dispatch this function call with `Root` origin?
+│  No 
+│
+⚙  pop call chain --pallet System --function remark --args "0x11" --url ws://localhost:9944/ --suri //Alice
+│  
+⚙  Encoded call data: 0x00000411
+│  
+◇  Do you want to submit the extrinsic?
+│  Yes 
+│
+◇  Extrinsic submitted with hash: "0xadbfbb2f92b22a2c5d6542ba80e4111b5c60057f594a6d155341879c5a46f96e"
+│
+└  Call complete.
+```
+
+#### Additional Options
+
+* If you have sudo rights and need to dispatch a function as Root origin you can specify it using the flag `--sudo`:
```

Comment:
I believe it is OK if we use more polkadot-sdk vocabulary here.

Something like:

To dispatch a call with the Root origin when the chain's runtime includes `pallet-sudo`, you can wrap the call in a `sudo.sudo()` call by using the `--sudo` flag.

Reply:
Good point! I agree, your approach is a better fit for the docs.